src/Reducer.js

```diff
@@ -207,6 +217,7 @@ export function reducer(
 ): ReduxState {
   switch (action.type) {
     case 'LOG':
+      console.debug(action.message)
```

Comment:
Message, not type?

Reply:
I suppose we need something more sophisticated than that, for the time being messages contains all required informations